README.md

```diff
@@ -1,2 +1,258 @@
-# metric-webapp
-React capstone project
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./banner.webp" alt="logo" />
+  <br/>
+
+  <h1><b>Metric-WebApp</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Metric-WEBAPP\] ](#-metric-webapp-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Metric-WEBAPP] <a name="about-project"></a>
+
+<!-- > Describe your project in 1 or 2 sentences. -->
+
+**[METRIC-WEBAPP]** This React capstone project (remember what they are?)is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+
+You will select an API that provides numeric data about a topic that you like and then build the webapp around it. The webapp will have several pages.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- > Describe the tech stack and include only the relevant sections that apply to your project. -->
+
```

Comment:
- [ ] Kindly remove all the commented out line from the README file to make it look more professional. It just a description to show how to make a professional README.

Reply:
noted, the line has been removed.